test/e2e/config/hivelocity-ci.yaml

```diff
@@ -119,8 +119,8 @@ variables:
   HIVELOCITY_SSH_PUB: "secret"
   HIVELOCITY_IMAGE_NAME: ${HIVELOCITY_IMAGE_NAME:-}
   HIVELOCITY_REGION: LAX2
-  HIVELOCITY_CONTROL_PLANE_MACHINE_TYPE: e2eControlPlane
-  HIVELOCITY_WORKER_MACHINE_TYPE: e2eWorker
+  HIVELOCITY_CONTROL_PLANE_MACHINE_TYPE: caphvlabel:deviceType=e2eControlPlane
```

Comment:
The value e2eControlPlane is hard-coded in the yaml file above.

I guess the env var is not needed any more.

If the env var is still needed, then I guess we should rename it to "..._LABEL" instead of "...TYPE". But talks with Janis about this first.

Reply:
@janiskemper ?

Reply:
I don't mind, we can follow what Thomas suggested